app/models/user.rb

```diff
@@ -564,11 +572,25 @@ def reindex_user_creations_after_rename
   end
 
   def add_renamed_at
+    return if User.current_user.is_a?(Admin)
```

Comment:
Conversely, there could also be confusion if there isn't an exception. For example
1. User is suspended
2. User is renamed
3. User is un-suspended within 7 days
4. User tries to rename and gets an error, even though they didn't rename themselves within the past few days.

I don't think that's particularly _common_, but I see that as a worse confusion than otherwise (especially since we log the action in the audits table either way)